examples/project-commands/flake.nix

```diff
@@ -0,0 +1,37 @@
+{
+  description = "Description for the project";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+
+    mission-control.url = "github:Platonic-Systems/mission-control";
+    flake-root.url = "github:srid/flake-root";
+  };
+
+  outputs = inputs@{ flake-parts, ... }:
+    flake-parts.lib.mkFlake { inherit inputs; } {
+      imports = [
+        inputs.mission-control.flakeModule
+        inputs.flake-root.flakeModule
+      ];
+      systems = [ "x86_64-linux" "aarch64-darwin" "x86_64-darwin" ];
+      perSystem = { config, self', inputs', pkgs, system, ... }: {
+        devShells.default = pkgs.mkShell {
+          nativeBuildInputs = with pkgs; [ avro-tools ];
+          inputsFrom = [ config.mission-control.devShell config.flake-root.devShell ];
+        };
+        mission-control = {
+          wrapperName = "run";
+          scripts = {
+            build = {
+              description = "convert files from .avdl to .avsc";
+              exec = ''
+                avro-tools idl2schemata "$FLAKE_ROOT/Hello.avdl" .
```

Comment:
`$FLAKE_ROOT/` here is redundant, since mission-control always runs commands from project root (unless `cdToProjectRoot` is turned off).

Reply:
I've been trying to remove it. But the moment I remove the script stops working.

```
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.io.File.toURI()" because the return value of "java.io.File.getParentFile()" is null
        at org.apache.avro.compiler.idl.Idl.<init>(Idl.java:53)
        at org.apache.avro.tool.IdlToSchemataTool.run(IdlToSchemataTool.java:47)
        at org.apache.avro.tool.Main.run(Main.java:67)
        at org.apache.avro.tool.Main.main(Main.java:56)
```

Reply:
Should be solved by Nix, fyi
- https://github.com/NixOS/nix/issues/8034

Reply:
What should we do then? Shall we leave it for now? I'm gonna follow the issue as well

Reply:
I think it's fine to leave as is for now.